src/connector/src/sink/console.rs

```diff
@@ -0,0 +1,200 @@
+            buffer: vec![],
+            prefix: config.prefix.unwrap_or_default(),
+            suffix: config.suffix.unwrap_or_default(),
+        })
+    }
+}
+
+#[async_trait]
+impl Sink for ConsoleSink {
+    async fn write_batch(&mut self, chunk: StreamChunk) -> Result<()> {
+        for (op, row_ref) in chunk.rows() {
+            let row_repr = join(row_ref.values().map(parse_datum), ",");
+            let op_repr = match op {
+                Op::Insert => "INSERT",
+                Op::UpdateDelete => "UPDATE_DELETE",
+                Op::UpdateInsert => "UPDATE_INSERT",
+                Op::Delete => "DELETE",
+            };
+            println!("{}{} [{}]{}", self.prefix, op_repr, row_repr, self.suffix);
+        }
+
+        Ok(())
+    }
+
+    async fn begin_epoch(&mut self, epoch: u64) -> Result<()> {
+        self.epoch = epoch;
+        Ok(())
+    }
+
+    async fn commit(&mut self) -> Result<()> {
+        println!(
+            "{}COMMIT [Epoch({})]{}",
+            self.prefix, self.epoch, self.suffix
+        );
+        Ok(())
+    }
+
+    async fn abort(&mut self) -> Result<()> {
+        Ok(())
+    }
+}
+
+fn parse_datum(datum: DatumRef<'_>) -> String {
+    match datum {
+        None => "NULL".to_string(),
+        Some(ScalarRefImpl::Int32(v)) => format!("Int32({})", v),
+        Some(ScalarRefImpl::Int64(v)) => format!("Int64({})", v),
+        Some(ScalarRefImpl::Float32(v)) => format!("Float32({})", v),
+        Some(ScalarRefImpl::Float64(v)) => format!("Float64({})", v),
+        Some(ScalarRefImpl::Decimal(v)) => format!("Decimal({})", v),
+        Some(ScalarRefImpl::Utf8(v)) => format!("Utf8({})", v),
+        Some(ScalarRefImpl::Bool(v)) => format!("Bool({})", v),
+        Some(ScalarRefImpl::NaiveDate(v)) => format!("Date({})", v),
+        Some(ScalarRefImpl::NaiveTime(v)) => format!("Time({})", v),
+        Some(ScalarRefImpl::Interval(v)) => format!("Interval({})", v),
+        _ => unimplemented!(),
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use std::sync::Arc;
+
+    use risingwave_common::array;
+    use risingwave_common::array::column::Column;
+    use risingwave_common::array::{ArrayImpl, I32Array, Op, StreamChunk, Utf8Array};
+    use risingwave_common::catalog::{Field, Schema};
+    use risingwave_common::types::DataType;
+
+    use crate::sink::console::{ConsoleConfig, ConsoleSink};
+    use crate::sink::Sink;
+
+    #[test]
+    fn test_console_sink() {
+        futures::executor::block_on(async {
+            let schema = Schema::new(vec![
+                Field {
+                    data_type: DataType::Int32,
+                    name: "id".into(),
+                    sub_fields: vec![],
+                    type_name: "".into(),
+                },
+                Field {
+                    data_type: DataType::Varchar,
+                    name: "name".into(),
+                    sub_fields: vec![],
+                    type_name: "".into(),
+                },
```

Comment:
suggest add support to Struct and Array type

Reply:
Added. The console format needs some additional tweaks other than `to_text()`, though.